src/components/new-request/search.vue

```diff
@@ -210,41 +225,59 @@
         </template>
 
         <!-- Numbered company bullets -->
-        <v-col v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY" cols="12">
-          <ul class="bullet-points">
-            <li>Your business name will be the Incorporation Number assigned by the Registry.</li>
-            <li>You can change your business name at a later date.</li>
-            <li>It is not possible to request a specific Incorporation Number.</li>
-          </ul>
-        </v-col>
+        <template v-if="selectedCompanyType === CompanyType.NUMBERED_COMPANY" cols="12">
+          <v-col v-if="isConversion && !isAlterOnline">
+            <div class="contact-registries">
+              To complete this alteration, please contact us at:
+              <p>
+                <br/>
+                  <v-icon small>mdi-phone</v-icon>  Canada and U.S. Toll Free:
```

Comment:
Don't use spaces for spacing. Use padding instead (or use the HTML entity `&nbsp;` once).

Reply:
Easy fix in this PR, please.

Reply:
It has been fixed, but it doesn't update at this page. It should be outdated.

Reply:
The latest files don't show the fix. Maybe you have a commit to push?